bazel/rules/rewrite_rpath/macos.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+PREFIX=$1
+INPUT=$2
+OUTPUT=$3
+
+cp "$INPUT" "$OUTPUT"
+install_name_tool -add_rpath "$PREFIX/lib" "$OUTPUT" 2>/dev/null || true
```

Comment:
Why do you suppress stderr? If something went wrong wouldn't it be useful information - or is it noise all the time, even on success?

Reply:
This is directly imported from https://github.com/DataDog/datadog-agent/blob/main/bazel/rules/replace_prefix.sh#L46-L62

My understanding is that if the expected rpath is already added to the binary, it would cause an error, so we want this for safety & idempotency, but this is definitely something we could discuss further